community/releases/release-manager-runbook/index.md

```diff
@@ -25,6 +35,7 @@ to merge outstanding changes by Monday:
 > please make sure they are merged by EOD next Monday. Once the branch is cut,
 > only fixes will be accepted into the release branches.
 
+
```

Comment:
What's the justification for the extra newlines in between sections? I think elsewhere we just have one newline above/below section headers, but could be wrong 🤷

Reply:
Yeah, we try not to include superfluous newlines and spaces. 

It's harmless though. Both our current md interpreter, Kramdown, and the one we'll probably be using soon, Goldmark, ignore extra newlines.

Reply:
For readability, since inside each section, they usually use 1 newline.

It gives some breathing room to the large document.